src/components/Navbar.tsx

```diff
@@ -105,7 +105,7 @@ function Navbar({ scrollToExplore }: { scrollToExplore: any }) {
             preview={false}
             style={{
               paddingRight: screens.md ? "1.5em" : "10px",
-              height: "26px",
+              height: "1.625em",
```

Comment:
Should we stick to one standard unit? I see `em` and `px` both getting used.

Reply:
yeah resolved! actually was testing the responsive edge cases